src/components/atoms/FormErrors.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { colors } from '../../style/common';
+
+import WobblyText from './WobblyText';
+
+const style = StyleSheet.create({
+  formErrors: {
+    padding: 10,
+  },
+  formErrorText: {
+    color: colors.red3,
+  },
+});
+
+interface FormErrorsProps {
+  errors: Array<string | undefined>;
+}
+const FormErrors: React.SFC<FormErrorsProps> = ({ errors }) => {
+  const filteredErrors = errors.filter((error) => !!error) as string[];
+  return (
+    <View style={style.formErrors}>
+      {filteredErrors &&
```

Comment:
this may have been code i wrote if you copied it over from the old repo (?), but looking at it now:

if it's the output of `errors.filter(...)`, then we shouldn't need the `filteredErrors &&` check first since it'll always be an array.

Reply:
It was copied over from the old repo! I've removed that bit.